src/lib.rs

```diff
@@ -71,21 +64,27 @@ pub fn render_tree(scene: &mut Scene, svg: &usvg::Tree) {
 pub fn render_tree_with<F: FnMut(&mut Scene, &usvg::Node) -> Result<(), E>, E>(
     scene: &mut Scene,
     svg: &usvg::Tree,
-    ts: &usvg::Transform,
     error_handler: &mut F,
 ) -> Result<(), E> {
-    render_tree_impl(scene, svg, &svg.view_box(), ts, error_handler)
+    render_tree_impl(
+        scene,
+        svg,
+        &svg.view_box(),
+        &usvg::Transform::identity(),
+        error_handler,
+    )
 }
 
+// A helper function to render a tree with a given transform.
 fn render_tree_impl<F: FnMut(&mut Scene, &usvg::Node) -> Result<(), E>, E>(
     scene: &mut Scene,
-    svg: &usvg::Tree,
+    tree: &usvg::Tree,
     view_box: &usvg::ViewBox,
     ts: &usvg::Transform,
```

Comment:
Is this ever not identity now?

Reply:
Yes.

`render_tree_impl` is called from `render_tree`, but it also gets called recursively for group nodes, which pass the parent transform down.